### DISCUSSION TITLE
Minimal scope requirements for Azure DevOps PAT?

### DISCUSSION THREAD

**Discussion by flcdrg** [renovatebot/renovate#19997] [Request Help]
What are the minimal scope requirements for Renovate for an Azure DevOps Personal Access Token?

I could use 'Full access' but using a more restricted scope would be preferred.

**Comment by rarkins** [comment]
It looks like this info is missing from https://docs.renovatebot.com/modules/platform/azure/

I'm not sure how granular Azure's PATs are. Renovate needs read/write access to code, issues, Pull Requests, and status checks.
